src/registry/api.py

```diff
@@ -198,10 +208,7 @@ def create_model(request, payload: ModelIn):
                     f"did you mean '{similar_lang}'?"
                 )
             }
-        else:
-            return 404, {
-                "message": f"Unknown language {payload.implementation_language}"
-            }
+        return 404, {"message": f"Unknown language {payload.implementation_language}"}
```

Comment:
Wouldn't be easier to have the implementation language as a select widget in the form?

Reply:
Good point, I think it would be way more intuitive also